irohad/ametsuchi/impl/flat_file/flat_file.hpp

```diff
@@ -117,34 +89,40 @@ namespace iroha {
 
       FlatFile &operator=(FlatFile &&rhs) = delete;
 
-      // ----------| private API |----------
-
-      /**
-       * Create storage in path with respect to last key
-       * @param last_id - maximal key written in storage
-       * @param path - folder of storage
-       */
-      FlatFile(Identifier last_id,
-               const std::string &path,
-               FlatFile::private_tag);
+      static constexpr size_t FIRST_BLOCK_AT{1};
```

Comment:
ALL_CAPS names are reserved for preprocessor macros. Please use kConstName for constants.